frontend_lib/src/css/Variable.styl

```diff
@@ -17,6 +17,9 @@ lightBlue = #3F9FF7
 
 offWhite = #fdfdfd
 
+json('../../../color.json')
+primaryColor = primary
```

Comment:
Will this work if the primary color is customized on an instance?

Reply:
frontend_lib needs to be recompiled after customization. Indeed, It's not great. We probably should find another way.